shared/src/vyxal/StringHelpers.scala

```diff
@@ -257,4 +257,14 @@ object StringHelpers:
       )
     )
 
+  def sentenceCase(s: String): String =
+    var b = true
```

Comment:
I prefer sentenceCase - that's what the element is called

Reply:
Oh, yeah - fair enough. It's just called `b` because I ported the code from [the Thunno 2 interpreter](https://github.com/Thunno/Thunno2/blob/main/src/thunno2/helpers.py#L2359) lol